packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -140,7 +140,6 @@ export default class Meetings extends WebexPlugin {
       this.meetingCollection.getByKey(CORRELATION_ID, MeetingsUtil.checkForCorrelationId(this.webex.internal.device.url, data.locus)) ||
       this.meetingCollection.getByKey(SIP_URI, data.locus.self && data.locus.self.callbackInfo && data.locus.self.callbackInfo.callbackAddress);
 
-      console.log('LOCUS EVENT ', JSON.stringify(data));
```

Comment:
Looks good to me . In future we might have to put more debug logs here for triaging . Sometimes due to a server event client might end call or stop media . To keep track of it we need to check the mercury event coming in

Reply:
We can use `debug()`. It's very helpful and controlled via command line environment variable.